ci/codefresh_cli.yml

```diff
@@ -1,5 +1,12 @@
 version: '1.0'
 steps:
+  Eslint:
```

Comment:
Why a new step and not after the unit tests?

Reply:
Single responsibility

Reply:
mmm ok, let's keep like that until we move to circle/github actions